src/display/ui/screens/ButtonLayoutScreen.cpp

```diff
@@ -186,11 +186,13 @@ void ButtonLayoutScreen::generateHeader() {
 
 	switch (Gamepad::resolveSOCDMode(gamepad->getOptions()))
 	{
-		case SOCD_MODE_NEUTRAL:               statusBar += " SOCD-N"; break;
-		case SOCD_MODE_UP_PRIORITY:           statusBar += " SOCD-U"; break;
-		case SOCD_MODE_SECOND_INPUT_PRIORITY: statusBar += " SOCD-L"; break;
-		case SOCD_MODE_FIRST_INPUT_PRIORITY:  statusBar += " SOCD-F"; break;
-		case SOCD_MODE_BYPASS:                statusBar += " SOCD-X"; break;
+		case SOCD_MODE_NEUTRAL:                         statusBar += " SOCD-N"; break;
+		case SOCD_MODE_UP_PRIORITY:                     statusBar += " SOCD-U"; break;
+		case SOCD_MODE_SECOND_INPUT_PRIORITY:           statusBar += " SOCD-L"; break;
+		case SOCD_MODE_FIRST_INPUT_PRIORITY:            statusBar += " SOCD-F"; break;
+		case SOCD_MODE_BYPASS:                          statusBar += " SOCD-O"; break;
```

Comment:
I changed this to SOCD-O because I wanted to use SOCD-X for the new one. Not sure if that's OK.